integration_tests/test_dbapi.py

```diff
@@ -355,8 +355,8 @@ def test_transaction_rollback(presto_connection_with_transaction):
         assert len(rows) == 1000
 
 
-def test_transaction_multiple(presto_connection_with_transaction):
-    with presto_connection_with_transaction as connection:
+def test_transaction_multiple(trino_connection_with_transaction):
+    with trino_connection_with_transaction as connection:
```

Comment:
Split this change into two commits. One with just the rename and the next one with the refactoring.

Thanks for catching this.

Reply:
Nevermind about this, I misread the diff.